executor/show_placement.go

```diff
@@ -128,23 +130,116 @@ func (e *ShowExec) fetchShowPlacementLabels(ctx context.Context) error {
 	return nil
 }
 
-func (e *ShowExec) fetchShowPlacement(_ context.Context) error {
-	err := e.fetchAllPlacementPolicies()
+func (e *ShowExec) fetchShowPlacementForTable(_ context.Context) (err error) {
```

Comment:
Privilege check?

Reply:
Issue tracked here: https://github.com/pingcap/tidb/issues/27977 . I'm considering implement privilege check in another pr.